From: Ben Hutchings <benh@debian.org>
Date: Mon, 25 Nov 2024 18:52:02 +0100
Subject: Revert "copy-firmware.sh: call ./check_whence.py before parsing the file"
Forwarded: not-needed

This reverts commit 6edd0fd36acb93c8075bbd700ad91f53fc810c0e.
check_whence.py will fail when run in firmware-nonfree, due to the
addition of the debian/ directory and the exclusion of some files.

---
--- a/copy-firmware.sh
+++ b/copy-firmware.sh
@@ -65,11 +65,6 @@ if test -d "$destdir"; then
     find "$destdir" -type d -empty >/dev/null || warn "destination folder is not empty."
 fi
 
-if test -e .git/config; then
-    $verbose "Checking that WHENCE file is formatted properly"
-    ./check_whence.py || err "check_whence.py has detected errors."
-fi
-
 # shellcheck disable=SC2162 # file/folder name can include escaped symbols
 grep -E '^(RawFile|File):' WHENCE | sed -E -e 's/^(RawFile|File): */\1 /;s/"//g' | while read k f; do
     test -f "$f" || continue
